components-core/src/main/java/com/adyen/checkout/components/core/internal/PermissionRequestParams.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2024 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ararat on 9/1/2024.
+ */
+
+package com.adyen.checkout.components.core.internal
+
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.core.internal.ui.PermissionHandlerCallback
+
+/**
+ * Runtime permission request data
+ */
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class PermissionRequestParams(
```

Comment:
We use `...Params` for the internal configuration objects, so this name might be a bit confusing. Maybe `PermissionRequestData` would solve this confusion? What do you think?